src/en/mayotune/src/eu/kanade/tachiyomi/extension/en/mayotune/MayoTune.kt

```diff
@@ -0,0 +1,15 @@
+package eu.kanade.tachiyomi.extension.en.mayotune
+
+import eu.kanade.tachiyomi.multisrc.mayotune.MayoTune
+import eu.kanade.tachiyomi.source.model.SManga
+
+class MayoTune : MayoTune("MayoTune", "https://mayotune.xyz", "en") {
```

Comment:
why make a multisrc if there is only one source in it?

Reply:
See the contributing guide. You can ask any questions in our discord

But basically, remove the multisrc and implement this class as httpsource directly. See another existing source for reference.